src/runtime/hvx_128.ll

```diff
@@ -35,6 +35,11 @@ define weak_odr <256 x i8> @halide.hexagon.interleave.vb(<256 x i8> %arg) nounwi
 
 declare <64 x i32> @llvm.hexagon.V6.vcombine.128B(<32 x i32>, <32 x i32>)
 
+define weak_odr <64 x i32> @halide.hexagon.concat_vectors(<32 x i32> %low, <32 x i32> %high) nounwind uwtable readnone alwaysinline {
```

Comment:
Remove this once the calling code is changed to use CodeGen_LLVM::concat_vectors.